lib/wait_for_healthy_container.rb

```diff
@@ -0,0 +1,25 @@
+require 'docker'
+require 'docker/compose'
+
+class WaitForHealthyContainer
+  def self.call(service:)
```

Comment:
This class/module looks a bit like it's caught in two identities where it's acting a bit as a class as a function (excuse the term I can't remember the proper one) with the `call` but then like a utility module with the other public methods. I wonder if we want to just rein it into being the latter and rename the class and call?